q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -1,35 +1,200 @@
+    });
+
+    toolbar.appendTo("#toolbar");
+
+    var category = $("#metadata-category .vega-bind");
+    category
+      .children(".vega-bind-name")
+      .replaceWith("<label> Metadata Category &nbsp;</label>");
+    category.children("select").addClass("form-control");
+
+    var sampling_depth_slider = $("#sampling-depth-slider .vega-bind");
+    $(sampling_depth_slider)
+      .find("label")
+      .remove();
+    $(sampling_depth_slider)
+      .find("input")
+      .attr("id", "rangeSlider");
+    $(sampling_depth_slider)
+      .find("input")
+      .attr("oninput", "updateSliderNum(this.value);highlighFeatureTable(this.value);");
+    $(sampling_depth_slider).prepend(
+      "<label> Sampling Depth &nbsp; </label> <br>"
+    );
+    $(sampling_depth_slider).append(
+      '<br> <input id="sliderNum" class="form-control" type="number" value="0" min="0" oninput="updateRangeSliderValue(this.value);;highlighFeatureTable(this.value);" />'
+    );
+    $(sampling_depth_slider).append("(Zero implies no even sampling.)");
+    sampling_depth_slider.children(".vega-bind-name").replaceWith("");
+
+    var rotateLabels = $("#rotate-labels .vega-bind");
+    rotateLabels
+      .children(".vega-bind-name")
+      .replaceWith("<label> Rotate Labels &nbsp;</label>");
+    rotateLabels.children("checkbox").addClass("form-control");
+  });
+</script>
+{% else %}
+<div class="row">
+  <div class="col-lg-4">
+    <div class="row">
+      <div class="col-lg-12" id="sampling-depth-slider">
+        <label> Sampling Depth &nbsp; </label> <br>
+        <input id="rangeSlider" class="form-control" type="range" min="0" max="{{max_count}}" value="0" class="slider" oninput="updateSliderNum(this.value);highlighFeatureTable(this.value);"> <br>
+        <input id="sliderNum" class="form-control" type="number" value="0" min="0" oninput="updateRangeSliderValue(this.value);highlighFeatureTable(this.value);" />
+        (Zero implies no even sampling.)
+      </div>
     </div>
-    <div id="histogram" class="col-lg-10 col-md-10 col-sm-12 col-xs-12"></div>
+
   </div>
-  <div class="row">
-    <div id="table" class="col-lg-10 col-md-10 col-sm-12 col-xs-12 pull-right"></div>
+
+</div>
+
+{% endif %}
+
+<div class="row">
+  <div class="col-lg-8">
+    <table id="feature-table" class="table table-striped">
+      <thead>
+        <tr>
+          <th scope="col">Sample ID</th>
+          <th scope="col">Feature Count</th>
+        </tr>
+      </thead>
+      <tbody id="table-body"></tbody>
+    </table>
   </div>
-  <script src='./dist/bundle.js'></script>
-  <script src='./data.jsonp'></script>
+</div>
+
+<script id="table-data" type="application/json">
+  {{ sample_frequencies_json }}
+</script>
+
+<script type="text/javascript">
+  var table_data = JSON.parse(document.getElementById("table-data").innerHTML);
+  // get object keys and store them in an ascending order based on the key value
+  sorted_samples = Object.keys(table_data).sort(function(a, b) {
```

Comment:
Should we make the sample ID be the tiebreaker in the case of two samples having the same count?